src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Color {
+    RED, GREEN, BLUE, YELLOW, ORANGE, PURPLE, BLACK, WHITE
+}
```

Comment:
here, please, write each value from a new line:
```suggestion
public enum Color {
    RED, 
    GREEN, 
    BLUE, 
    YELLOW, 
    ORANGE, 
    PURPLE, 
    BLACK, 
    WHITE
}
```